### ISSUE TITLE
Daydream Rendering Corruption Flicker on One Eye

### ISSUE THREAD

**Issue by DWishR** [googlevr/gvr-unity-sdk#896]
Summary:
We're tracking an issue where rendering gets corrupted for at least one eye on the daydream platform. I usually see it on the left eye, and it flickers back and forth to a true frame. I'm attaching a screenshot of the issue. The best repro note that we have is to task switch into and out of the game repeatedly until the issue presents itself. It does occasionally occur without warning during normal playthrough of another project we have. With the attached project, it seems easier to reproduce with a Development Build but we see it occur in all build configurations.

We see this in version 2017.1.3p3 (and every 2017.1 we've tried). The issue appears to be resolved in 2017.3.1p4 although we don't see anything in those specific patch notes that would indicate a fix for this (it would be helpful if we knew precisely which patch fixed it, and we probably need to test it more to make sure). We would greatly appreciate a fix in 2017.1 or being pointed out how to patch in a fix to 2017.1.

I've got a ticket inside FogBugz that hasn't made it to the Issue Tracker. It's Case 1025144.

Found using:
 * Google VR SDK version: Google VR NDK 1.130 (from patch notes)
 * Unity version: 2017.1.3p3
 * Phone manufacturer, model, and O/S version: Google Pixel, Android 8.1.0, OPM2.171019.029
 * Viewer manufacturer & model: Google Daydream View (original)

Steps to reproduce the issue:
Make a development build and run it on an e.g. Google Pixel. Once you are in Daydream and the spinning cube appears:
    A) Drag your finger in from the edge of the phone (on the side where the charging port is) to reveal the home button
    B) Press the home button
    C) Press the task selection button (usually a square) on the android device to pull up running applications
    D) Press on this application (Daydream Issue) to bring it on screen
    E) Note the corruption, or return to A and repeat until it appears
It's easiest if you configure the "Developer Options" within the Daydream menu (available by pressing the gear on the top right of the screen while the app is running, and then tapping "Build Version" repeatedly until Developer Options appears) to "Skip VR entry screens" in order to enable fast entering and exiting of the application without having to put the device into and out of a headset.

Workarounds:
None known

Additional comments:

![screenshot_20180411-151653](https://user-images.githubusercontent.com/10715721/38696699-c8d0a14e-3e44-11e8-8b85-7f843e9807e1.png)
[Daydream_Issue.zip](https://github.com/googlevr/gvr-unity-sdk/files/1904487/Daydream_Issue.zip)

**Comment by DWishR** [comment]
I forgot I was asked to add the version of VRCore on the device:
```
>adb shell pm dump com.google.vr.vrcore | grep version
      versionCode=160733190 minSdk=19 targetSdk=26
      versionName=1.14.190701955
      versionCode=160726230 minSdk=19 targetSdk=25
      versionName=1.6.156794279
```

**Comment by DWishR** [comment]
I just reprod this with 2017.3.1p3 which makes it much scarier/urgent. It appears to have a slightly different effect on that version.

Opened case 1025479 in FogBugz.

![screenshot_20180412-115305](https://user-images.githubusercontent.com/10715721/38700575-61622e9a-3e50-11e8-9ff8-03704221db6b.png)
![screenshot_20180412-115256](https://user-images.githubusercontent.com/10715721/38700576-617aa358-3e50-11e8-9b8b-995bf617e8c9.png)

**Comment by rusmaxham** [comment]
The left eye flicker is fixed in 2017.2 and newer.  This is the only fix available at this time.  It will not be backported to 2017.1.
